Userland/Libraries/LibWeb/HTML/EventNames.cpp

```diff
@@ -18,7 +18,7 @@ ErrorOr<void> initialize_strings()
     VERIFY(!s_initialized);
 
 #define __ENUMERATE_HTML_EVENT(name) \
-    name = #name;
+    name = TRY(FlyString::from_utf8(StringView { #name, strlen(#name) }));
```

Comment:
Still seems overly complicated - just `from_utf8(#name##sv)` is fine, no?

Reply:
Can also do `TRY(#name##_fly_string)` as we did here:

https://github.com/SerenityOS/serenity/blob/8a8340b3cdda0ab0dfd1dad734db933400422cfa/Userland/Libraries/LibWeb/DOM/MutationType.cpp#L20

Reply:
That's much nicer. Updated